src/Test/Perf/Tests/projects/csharp/CSharpCompilerNoAnalyzer.csx

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
This is a *lot* of duplicated code.

Please revert this back to a for-loop going over each rsp file.